RELEASE-NOTES.txt

```diff
@@ -6,7 +6,7 @@
 -----
 * [*] a11y: VoiceOver has been improved on the Menus view and now announces changes to ordering. [#18155]
 * [*] Notifications list: remove comment Trash swipe action. [#18349]
-* [*] Reader: Fixed a bug that caused cut off content in reader web view [#16106]
+* [*] Web previews now abide by safe areas when a toolbar is shown [#18127]
```

Comment:
Is that change expected (especially since that does not seem like just a wording update of a single item, but instead removal of an item related to one PR #16106 while adding a different item related to a different PR #18127)?

And do we need to inform our freelance writer (Grace) about that change if so?

Reply:
👋 @AliSoftware - [this PR contains the backstory of this change](https://github.com/wordpress-mobile/WordPress-iOS/pull/18401).

Summary:
- The work started out as a fix for https://github.com/wordpress-mobile/WordPress-iOS/issues/16106 by an external contributor
- During the review process we made them aware that the fix had broader scope than just reader
- At the tail end (a little too late) I updated their contributed release note and revised their PR title to better reflect the changes. I then created https://github.com/wordpress-mobile/WordPress-iOS/pull/18401 to hopefully prevent any surprises on the release side.

Should I have included the Owl Team on that PR to loop everyone in? Any tips welcome! 🙇

Reply:
Thanks for the helpful extra context @twstokes !

I think it's always a good idea to ping your Release Manager especially when you change either release notes (like here) or strings (like for the case of the other review comment I left) in a PR that lands in `release/*` branch after the code-freeze.

Both release notes and strings are frozen during code freeze and we then use those frozen versions to kick off the rest of the process (ask our freelance writer to write nice notes from this frozen draft, send frozen strings to our translation vendor, …), which is why any change to those after code freeze is important to signal to your RM so they can adjust for those last-minute tweaks appropriately 🙂.

---

That being said, given the additional context you shared here, I think the wording that ended up being used in the Editorialized version of the Release Notes [here below](https://github.com/wordpress-mobile/WordPress-iOS/pull/18411/files#diff-a0ae2ed3128fff2f7c059edd7fe0d51f6a7f1ec9ecea83b6d46f6b8d16a7d3bcR7) is probably OK and matches what ultimately landed?

Reply:
Thanks @AliSoftware, that makes sense!

> I think the wording that ended up being used in the Editorialized version of the Release Notes [here below](https://github.com/wordpress-mobile/WordPress-iOS/pull/18411/files#diff-a0ae2ed3128fff2f7c059edd7fe0d51f6a7f1ec9ecea83b6d46f6b8d16a7d3bcR7) is probably OK and matches what ultimately landed?

I agree!